qemu/tests/hugepage_mem_stress.py

```diff
@@ -29,12 +34,26 @@ def heavyload_install():
             install_cmd = params["install_cmd"] % winutil_drive
             session.cmd(install_cmd)
 
+    if params.get("non_existent_point"):
+        error_context.context("This path %s, doesn't mount hugepage." %
+                              params.get("non_existent_point"), logging.info)
+        if os.path.ismount("%s" % params.get("non_existent_point")):
+            process.run("umount %s" % params.get("non_existent_point"))
+        params["start_vm"] = "yes"
+        env_process.preprocess_vm(test, params, env, params["main_vm"])
+
     os_type = params["os_type"]
     verify_wait_timeout = params.get_numeric("verify_wait_timeout", 60)
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     session = vm.wait_for_login()
 
+    if params.get("non_existent_point"):
+        error_context.context("Check large memory pages free on host.",
+                              logging.info)
+        if utils_memory.get_num_huge_pages() != utils_memory.get_num_huge_pages_free():
```

Comment:
Had better move the check after stress test. BTW, why is the check only needed when using non-exist point?

Reply:
To keep the original case structure.
Because of the original case: there is no such test point in the "stress test"

Reply:
If in existent mem-path, HugePages will be used.
So no need to do this check.